Qs.Presentation.WebApi/Dockerfile

```diff
@@ -0,0 +1,20 @@
+﻿FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /src
+COPY ["QSWebAPI.csproj", "./"]
+RUN dotnet restore "QSWebAPI.csproj"
```

Comment:
Don't forget to update these names, the csproj file has been moved from `QSWebAPI.csproj` to `Qs.Presentation.WebApi/Qs.Presentation.WebApi.csproj`, and finally make sure to test if the Dockerfile works. If you don't want to spend the time, it's also fine to exclude it from the repo for now.

Reply:
Excluded